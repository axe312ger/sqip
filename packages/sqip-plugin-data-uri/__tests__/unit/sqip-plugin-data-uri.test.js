// import sqipPluginDatauri from '../../src/sqip-plugin-datauri'

describe('sqip-plugin-datauri', () => {
  it('needs tests', () => {})
})
