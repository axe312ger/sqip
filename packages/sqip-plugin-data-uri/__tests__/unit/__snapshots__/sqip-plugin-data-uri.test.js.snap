// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sqip-plugin-data-uri encodes svg with base64 1`] = `"data:image/svg+xml;base64,PHN2ZyAvPg=="`;
